app/models/membership_request.rb

```diff
@@ -8,6 +8,8 @@ class MembershipRequest < ActiveRecord::Base
 
   validates :group, presence: true
 
+  validates_length_of :introduction, maximum: 250
```

Comment:
I'm nervous about this - there's about 5000 records in the database that have more than 250 sized introductions. What will happen to them?